core/src/main/java/hudson/WebAppMain.java

```diff
@@ -254,7 +254,10 @@ public Locale get() {
             // check that and report an error
             try {
                 File f = File.createTempFile("test", "test");
-                f.delete();
+                boolean result = f.delete();
+                if (!result) {
+                    LOGGER.log(FINE, "Temp file test.test could not be deleted.");
```

Comment:
Should it throw exception here? I read the comment, and it looks like it would be safer to report the error